go/chat/globals/globals.go

```diff
@@ -35,6 +36,7 @@ type ChatContext struct {
 	LiveLocationTracker  types.LiveLocationTracker        // track live location messages for updates
 	BotCommandManager    types.BotCommandManager          // manages commands from bots in convs
 	UIInboxLoader        types.UIInboxLoader              // manages loading inbox for UI
+	Badger               *badges.Badger                   // app badging
```

Comment:
How come you did this?

Reply:
needed access in sender.go and server.go, thought it was easier to hang off of g